components/nds-prev-next/src/PrevNext.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import "./../scss/prev-next.scss";
+
+const Link = ({ text, destination, elementType: LinkTag = "a", intro }) => {
+	let linkProps = {
+		className: "prev-next__link"
+	};
+	if (LinkTag === "a") {
+		linkProps.href = destination;
+	} else {
+		linkProps.to = destination;
+	}
+	return (
+		<LinkTag {...linkProps}>
+			<span className="prev-next__link-intro">{intro}</span>
+			<span className="prev-next__link-text">{text}</span>
```

Comment:
The `text` prop in the TS def is optioanl, in which case should we conditionally render this span?

Reply:
There'll always need to be text of some kind - at this point the text is at least the destination